### PR TITLE
chore: Defaults lazyloading to true

### PR DESCRIPTION
**Pull request by claudiachua**
Internal JIRA ticket: [DI-2059
](https://issues.corp.twilio.com/browse/DI-2059)

### CODE REVIEW

**Review by JenniferMah — APPROVED** [review]

**Comment by childish-sambino** [comment]
nit: Should drop the `[DI-2059]` from public repos so it doesn't get in the changelog.

File: lib/rest/Twilio.js [hunk 1]
```diff
@@ -127,6 +127,9 @@ function Twilio(username, password, opts) {
   this.password = password || env.TWILIO_AUTH_TOKEN;
   this.accountSid = opts.accountSid || this.username;
   this._httpClient = opts.httpClient;
+  if (opts.lazyLoading == undefined) {
+    opts.lazyLoading = true;
+  }
   if (!opts.lazyLoading) {
```
Comment (childish-sambino) [inline lib/rest/Twilio.js:133]: nit: Could simplify to just:
```node
if (opts.lazyLoading === false) {
   this._httpClient = this.httpClient;
}
```

Comment (claudiachua) [reply lib/rest/Twilio.js:133]: I assume the same change will apply to Line 187 too?

Comment (childish-sambino) [reply lib/rest/Twilio.js:133]: Yes.

**Pull request merged**